assets/src/edit-story/app/api/apiProvider.js

```diff
@@ -205,17 +205,14 @@ function APIProvider({ children }) {
     [link]
   );
 
-  const getAllFonts = useCallback(
-    ({}) => {
-      return apiFetch({ path: fonts }).then((data) =>
-        data.map((font) => ({
-          value: font.name,
-          ...font,
-        }))
-      );
-    },
-    [fonts]
-  );
+  const getAllFonts = useCallback(() => {
+    return apiFetch({ path: fonts }).then((data) =>
```

Comment:
This will be good. I noticed some Safari did not like destructuring an object with empty params.